webui/src/services/telemetryservice.js

```diff
@@ -0,0 +1,41 @@
+export default class TelemetryService {
+    constructor() {
+    }
+
+    setUser(user) {
+        window.appInsights.setAuthenticatedUserContext(user.id);
+        // this.sha256(user.id).then(digest => {
+        //     window.appInsights.context.user.id = digest;
+        // })
```

Comment:
Is it very important comment?

Reply:
Why you think we don't need userid?

Reply:
`setAuthenticatedUserContext` accepts anything unique - id, username etc., but it does not send it over the wire to Azure